app.js

```diff
@@ -9,6 +10,21 @@ app.use(require("body-parser").urlencoded({ extended: true }));
 
 require("dotenv").config(); // to load the .env file into the process.env object
```

Comment:
Just to be safe, you should place this line as close to the top of your file as possible. No need to make any changes now; just something to bear in mind.